src/Popover/Popover.test.js

```diff
@@ -25,37 +24,82 @@ const PopoverComponent = () => (
             },
           ]}
         />
-      </Fragment>
+      </div>
     }
   >
-    <Button>click me</Button>
+    <button>click me</button>
   </Popover>
 )
 
+const defaultWaitTimeMs = 500
+
+const waitExpect = (callback, timeMs = defaultWaitTimeMs) =>
+  wait(callback, { timeout: timeMs })
+
 describe('Popover', () => {
-  it('should render popover when button is clicked', () => {
-    const component = mount(<PopoverComponent />)
+  afterEach(cleanup)
+
+  it('should render popover when button is clicked', async () => {
+    const { container, getByText } = render(popoverComponent)
+    const button = getByText('click me')
+    let content = container.querySelector('#content')
+    expect.assertions(2)
 
-    component.simulate('click')
-    expect(component.find('div > div').first()).toHaveLength(1)
+    expect(content).toBeNull()
+
+    // open popover
+    fireEvent.click(button)
+
+    content = await waitForElement(() => container.querySelector('#content'))
+
+    expect(content).not.toBeNull()
   })
 
-  it('should remove popover when button is clicked', () => {
-    const component = mount(<PopoverComponent />)
+  it('should remove popover when button is clicked', async () => {
+    const { container, getByText } = render(popoverComponent)
+    const button = getByText('click me')
+    let content = container.querySelector('#content')
+
+    expect(content).toBeNull()
 
-    component.simulate('click')
-    component.simulate('click')
+    // open popover
+    fireEvent.click(button)
 
-    expect(component.find('div > div').first()).toHaveLength(0)
+    content = await waitForElement(() => container.querySelector('#content'))
+
+    expect(content).toBeDefined()
+
+    // close popover
+    fireEvent.click(button)
+
+    await waitExpect(
```

Comment:
esse await poderia ser diretamente, sem a necessidade de definir a função `waitExpect`
```suggestion
await wait(
      () => expect(container.querySelector('#content')).toBeNull()
    )
```

Reply:
This rename make sense because this function put a default value to the options of the (`wait` function)[https://github.com/kentcdodds/react-testing-library#wait] from `react-testing-library`. Also the function `wait` will wait until the callback dosn't throw an error, in this case it will wait until the expect condition is complete. 
Try to put a limit to the `expect` calls and you will see that the `expect` is called many times.
```suggestion
    expect.assertions(2)
    await waitExpect(
```

Reply:
Yeap this is the reason for this change, I don't wanna wait for too long to an animation to run. Imane a lot of tests waiting for 4.5s soon our tests will become slower than the API tests.

Reply:
é que ele so vai demorar 4.5s se o expect não resolver, ou seja, se a animção for de fato longa, senão ele retorna antes. Mas ok